app/javascript/components/project_members/NewProjectMember.vue

```diff
@@ -73,4 +138,30 @@ export default {
   padding: 1em;
   margin: 1em;
 }
+
+#RoleDivider {
+  margin-top: 0.25rem;
+}
+
+.role-row {
+  margin-bottom: 1rem;
+}
+
+.role-input {
+  margin-left: 0rem;
+  margin-right: 1rem;
+  margin-top: 0rem;
```

Comment:
should use bootstrap spacing utilities where possible https://getbootstrap.com/docs/4.0/utilities/spacing/